.goreleaser.yaml

```diff
@@ -107,6 +107,34 @@ docker_signs:
     - '${artifact}'
     - "--yes" # needed on cosign 2.0.0+
 
+brews:
```

Comment:
We need to make sure the `GORELEASER_GITHUB_TOKEN` in the CI/CD has access to commit to the `openfga/homebrew-tap` repository.

Reply:
`GORELEASER_GITHUB_TOKEN` has been added